projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -300,11 +280,9 @@ public function add_comments_menu( $wp_admin = false ) {
 	/**
 	 * Adds Appearance menu.
 	 *
-	 * @param bool $wp_admin_themes Optional. Whether Themes link should point to Calypso or wp-admin. Default false (Calypso).
-	 * @param bool $wp_admin_customize Optional. Whether Customize link should point to Calypso or wp-admin. Default false (Calypso).
 	 * @return string The Customizer URL.
 	 */
-	public function add_appearance_menu( $wp_admin_themes = false, $wp_admin_customize = false ) {
+	public function add_appearance_menu() {
```

Comment:
This is a tricky menu. Except for the `themes` menu it doesn't have pure Calypso equivalents, it does though have WP Admin pages vs Customizer panels.

eg `/wp-admin/nav-menus.php` vs `https://wordpress.com/customize/[DOMAIN]?autofocus%5Bpanel%5D=nav_menus`

Also, in case of Atomic sites we want to always (regardless of any choice) point to WP Admin customizer cause the iFrame - redirection doesn't work really well.

Do you think that maybe in case of `add_appearance_menu` we could keep `$wp_admin_customize = false` so that we are sure we don't introduce any regressions? We can follow up later and see if we can make simplify it further.

Reply:
I preferred to keep the current behavior in order to make both the testing and review easy. Maybe we can look into changing this in a follow-up issue?

Reply:
Nevermind, I just saw that you have created different versions of `get_preferred_view` for the different environments.